WordPress/twentytwentyfour
# Fix/services image duotone

**Description**

Addresses the Services pattern image mentioned in #199 

**Screenshots**

![Screenshot 2023-09-08 at 10 55 35 AM](https://github.com/WordPress/twentytwentyfour/assets/405912/17a488c9-e7ec-4716-a36a-bcf7f2e3ee63)
![Screenshot 2023-09-08 at 10 55 55 AM](https://github.com/WordPress/twentytwentyfour/assets/405912/eb7cad41-7331-4c0d-aaa2-eeb43cfba115)


**Testing Instructions**

1. Add the 'Services - Call To Action' pattern and verify image is matching Figma and black-and-white duotone is applied
